app/domain/operations/households/deactivate_financial_assistance_eligibility.rb

```diff
@@ -30,7 +30,7 @@ def validate(params)
       # returns Success(nil)
       def execute(values)
         family = Operations::Families::Find.new.call(id: BSON::ObjectId(values[:family_id]))
-        tax_households = family.success.active_household.latest_active_tax_households
+        tax_households = family.success.active_household.latest_active_tax_households.tax_household_with_year(values[:date].to_date.year)
```

Comment:
End dating active tax households for a specific year instead of all active tax households.